examples/mining_ar.py

```diff
@@ -0,0 +1,128 @@
+              '{:1.2f}'.format(ar.confidence), COLOR_CODES['default'], end='\t')
+        print(ar.left, '->', ar.right)
+
+
+def print_itemnames(itemnames):
+    print(f'{COLOR_CODES["blue_bg"]}Total number of items:'
+          f'{COLOR_CODES["default_bg"]} {len(itemnames)}')
+    print(*itemnames, sep='\n')
+
+
+def scenario_tabular():
+    algo = desbordante.ar.algorithms.Default()
+    algo.load_data(table=(TABLE_TABULAR, ',', True), input_format='tabular')
+    algo.execute(minconf=1)
+    table = pandas.read_csv(TABLE_TABULAR, header=None)
+
+    print("As the first example, let's look at the dataset containing receipts "
+          'from some supermarket using input_format="tabular". In this '
+          'format, each table row lists all items participating in the same '
+          'transaction. Note that in this table some rows may be shorter than others.\n')
+    print_table(table)
+    print("\nLet's see the first 10 association rules (ARs) that are present "
+          'in the dataset with minconf=1. If no minsup is specified, '
+          'the default value of minsup=0 is used.\n')
+    print_ars(algo.get_ars())
+    print("\n['Eggs'] -> ['Milk'] with confidence 1 means that whenever eggs "
+          'are found in the receipt, milk will '
+          f'{COLOR_CODES["green"]}always{COLOR_CODES["default"]} '
+          'be present as well. The same holds true for all other rules with '
+          f'{COLOR_CODES["bold_green"]}confidence 1{COLOR_CODES["default"]}.')
+
+    print("\n\nNow, let's examine the same dataset with "
+          f'{COLOR_CODES["yellow"]}minconf=0.7{COLOR_CODES["default"]}')
+    algo.execute(minconf=0.7)
+    print_ars(algo.get_ars())
+    print("\n['Milk'] -> ['Eggs'] with confidence 0.75 means that when milk "
+          'is found in the receipt, the chance of eggs being '
+          'present amounts to 75%. So, customers are '
+          f'{COLOR_CODES["bold_yellow"]}likely{COLOR_CODES["default"]} to buy eggs '
+          f'with milk, but {COLOR_CODES["bold_red"]}not always.{COLOR_CODES["default"]}')
+
+    print('\nLet us turn to the next issue. You can observe that there are too many '
+          'association rules found in this small dataset. This happens because we did not '
+          'set up the support value. Due to this fact, the system discovers '
+          'all association rules, even those which are present only '
+          'once in the dataset. Now let\'s see the results with '
+          f'{COLOR_CODES["yellow"]}minsup=0.5{COLOR_CODES["default"]} and '
+          f'{COLOR_CODES["yellow"]}minconf=0.5{COLOR_CODES["default"]}.\n')
+    algo.execute(minsup=0.5, minconf=0.5)
+    print_ars(algo.get_ars())
+    print('\nNow you can see that the number of association rules have decreased '
+          'significantly. This happened due to minsup being set to 0.5, which basically '
+          'means that association rule needs to are present in 50% of all transactions '
```

Comment:
Adding a definition to the support value line would necessitate changes here to remove redundant information